README.md

```diff
@@ -53,23 +55,33 @@ and processor architecture listed below.
 * CentOS 8
 * Debian 9
 * Debian 10
-* FreeBSD 11.3
+* FreeBSD 11.4
 * FreeBSD 12.1
-* Ubuntu 16.04
-* Ubuntu 18.04
+* Ubuntu 16.04.7
+* Ubuntu 18.04.5
+* Ubuntu 20.04.1
 
 ### Supported database engine versions
 
-Operating System | MySQL/MariaDB   | PostgreSQL
+Operating System | MariaDB         | PostgreSQL
 ---------------- | ----------------| -----------
-CentOS 7         | 5.6 (MySQL)     | 9.3
-CentOS 8         | 8.0 (MySQL)     | 10.6
-Debian 9         | 10.1 (MariaDB)  | 9.6
-Debian 10        | 10.3 (MariaDB)  | 11.7
-FreeBSD 11.3     | 5.7 (MySQL)     | 11.7
-FreeBSD 12.1     | 5.7 (MySQL)     | 11.7
-Ubuntu 16.04     | 10.0 (MariaDB)  | 9.5
-Ubuntu 18.04     | 10.1 (MariaDB)  | 10.12
+CentOS 7         | 5.5 (#)         | 9.3 (##)
+CentOS 8         | ????            | 10.6 (##)
+Debian 9         | 10.1            | 9.6
+Debian 10        | 10.3            | 11.7
+FreeBSD 11.4     | 10.5.5 (###)    | 12.4
+FreeBSD 12.1     | 10.5.5 (###)    | 12.4
+Ubuntu 16.04     | 10.0            | 9.5
+Ubuntu 18.04     | 10.1            | 10.12
+Ubuntu 20.04     | 10.3 (#)        | 12.4 (#)
+
+###) The latest stable in the highest release serie. There is no
```

Comment:
I'm not sure I agree with the choice of syntax here.

Reply:
Me neither. They are only meant to be in the draft version of the file, not the final version for the release. I wrote them for you, the reviewer of this PR.